packages/devtools_app/lib/src/debugger/key_sets.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import '../config_specific/host_platform/host_platform.dart';
+
+final LogicalKeySet focusLibraryFilterKeySet = LogicalKeySet(
+  HostPlatform.instance.isMacOS
+      ? LogicalKeyboardKey.meta
+      : LogicalKeyboardKey.control,
+  LogicalKeyboardKey.keyP,
+);
+
+final LogicalKeySet goToLineNumberKeySet = LogicalKeySet(
+  HostPlatform.instance.isMacOS
+      ? LogicalKeyboardKey.meta
+      : LogicalKeyboardKey.control,
+  LogicalKeyboardKey.keyG,
+);
+
+final LogicalKeySet searchInFileKeySet = LogicalKeySet(
+  HostPlatform.instance.isMacOS
+      ? LogicalKeyboardKey.meta
+      : LogicalKeyboardKey.control,
+  LogicalKeyboardKey.keyF,
+);
+
+final LogicalKeySet escapeKeySet = LogicalKeySet(
+  LogicalKeyboardKey.escape,
+);
+
+final LogicalKeySet openFileKeySet = LogicalKeySet(
+  HostPlatform.instance.isMacOS
+      ? LogicalKeyboardKey.meta
+      : LogicalKeyboardKey.control,
+  LogicalKeyboardKey.keyP,
```

Comment:
this is the same as `focusLibraryFilterKeySet` above. Maybe use 'o' for this one for now? At least until @bkonyi's changes land and we have ripped out the filter in the libraries window. Then we can decide whether we want to have this hot key be cmd + p or cmd + o long term. I expect both the open file feature and Ben's changes will be done for the Oct release.

Reply:
Oh good catch! Switched to cmd + o

Reply:
CTRL-P is definitely preferable since most editors use that as a shortcut for file searching. Will it help if I upload a small PR to remove the filter box? It shouldn't be very difficult to do.

Reply:
Sure, if that's not too much trouble that would be great! Thanks

Reply:
https://github.com/flutter/devtools/pull/3319

Reply:
Thank you!

Reply:
commented on Ben's PR, but will add a note here too - we shouldn't land #3319 until after the release goes out tomorrow. We don't want to remove the filtering functionality before we replace it with the open file dialog feature.